docs/development/extensions.md

```diff
@@ -72,7 +72,7 @@ All of these community extensions can be downloaded using [pull-deps](../operati
 |Name|Description|Docs|
 |----|-----------|----|
 |ambari-metrics-emitter|Ambari Metrics Emitter |[link](../development/extensions-contrib/ambari-metrics-emitter.md)|
-|druid-azure-extensions|Microsoft Azure deep storage.|[link](../development/extensions-contrib/azure.md)|
+|druid-azure-extensions|Microsoft Azure deep storage.|[link](extensions-core/azure.md)|
```

Comment:
This should be moved up into the core extensions section instead of the contrib section

Reply:
fixes